### ISSUE TITLE
0.4.4.2: check_mem: negative values

### ISSUE THREAD

**Issue by lr1980** [mickem/nscp#191]
```
check_memory
L        cli CRITICAL: CRITICAL: committed = 5.266GB, physical = 5.067GB
L        cli  Performance data: 'committed'=1.2661GB;0.74964;-1.65664;0;15.93706 'committed %'=7%;4;-10;0;100 'physical'=1.06715GB;-0.80035;0.5996;0;13.99956 'physical %'=7%;-5;4;0;100
````

I think the check not working properly!?

**Comment by mickem** [comment]
I would upgrade to 0.4.3 as the core checks are no longer using PDH...

**Comment by mickem** [comment]
This has been fixed in 0.5.0

**Comment by mickem** [comment]
Fixed in 0.5.0.16: https://www.nsclient.org/download/0.5.0/
